### PR TITLE
New fork changes

### PR DESCRIPTION
**Pull request by drizuid**
undeprecate and update to new fork, remove arm32
please verify the jenkins crap

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # syntax=docker/dockerfile:1
 
-FROM ghcr.io/linuxserver/baseimage-alpine-nginx:3.17
+# update to alpine 3.18 from archived linuxserver/docker-cops version
```
Comment (nemchik) [inline Dockerfile:3]:
```suggestion
```
Comment isn't needed

File: Dockerfile [hunk 2]
```diff
@@ -11,36 +12,40 @@ LABEL maintainer="chbmb"
 
 RUN \
   echo "**** install runtime packages ****" && \
+  # update to PHP 8.2 from archived linuxserver/docker-cops version
```
Comment (nemchik) [inline Dockerfile:15]:
```suggestion
```
Comment isn't needed

File: Dockerfile [hunk 3]
```diff
@@ -11,36 +12,40 @@ LABEL maintainer="chbmb"
...
-    php81-opcache \
-    php81-phar \
-    php81-pdo_sqlite \
-    php81-zip && \
+    # libxml2 \
+    php82-dom \
+    php82-gd \
+    php82-intl \
+    # php82-json \
+    php82-mbstring \
```
Comment (nemchik) [inline Dockerfile:22]:
```suggestion
```
This is already included in the base https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/master/Dockerfile

File: Dockerfile [hunk 4]
```diff
@@ -11,36 +12,40 @@ LABEL maintainer="chbmb"
...
-    php81-intl \
-    php81-opcache \
-    php81-phar \
-    php81-pdo_sqlite \
-    php81-zip && \
+    # libxml2 \
+    php82-dom \
+    php82-gd \
+    php82-intl \
+    # php82-json \
```
Comment (nemchik) [inline Dockerfile:21]:
```suggestion
```
This is already included in the base https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/master/Dockerfile

File: Dockerfile [hunk 5]
```diff
@@ -11,36 +12,40 @@ LABEL maintainer="chbmb"
...
   curl -o \
     /tmp/cops.tar.gz -L \
-    "https://github.com/seblucas/cops/archive/${COPS_RELEASE}.tar.gz" && \
+    "https://github.com/mikespub-org/seblucas-cops/archive/${COPS_RELEASE}.tar.gz" && \
   mkdir -p \
     /app/www/public && \
   tar xf /tmp/cops.tar.gz -C \
     /app/www/public --strip-components=1 && \
   cd /app/www/public && \
+  # use standard composer 2.x now, no need to install older 1.x version
```
Comment (nemchik) [inline Dockerfile:46]:
```suggestion
```

File: jenkins-vars.yml [hunk 6]
```diff
@@ -5,10 +5,10 @@ project_name: docker-cops
 external_type: github_stable
 release_type: stable
 release_tag: latest
-ls_branch: master
+ls_branch: mikespub
 repo_vars:
   - EXT_GIT_BRANCH = 'master'
-  - EXT_USER = 'seblucas'
+  - EXT_USER = 'mikespub'
   - EXT_REPO = 'cops'
```
Comment (nemchik) [inline jenkins-vars.yml:12]:
```suggestion
  - EXT_USER = 'mikespub-org'
  - EXT_REPO = 'seblucas-cops'
```

**Review by nemchik — CHANGES_REQUESTED** [review]
I didn't add duplicate comments to the arm64 dockerfile but it should all apply there too.

**Comment by drizuid** [comment]
> I didn't add duplicate comments to the arm64 dockerfile but it should all apply there too.

Thanks!!

File: Dockerfile [hunk 7]
```diff
@@ -1,6 +1,4 @@
-# syntax=docker/dockerfile:1
```
Comment (nemchik) [inline Dockerfile:1]: This comment is needed

File: Dockerfile.aarch64 [hunk 8]
```diff
@@ -1,6 +1,4 @@
-# syntax=docker/dockerfile:1
```
Comment (nemchik) [inline Dockerfile.aarch64:1]: This comment is needed

**Review by nemchik — CHANGES_REQUESTED** [review]
Add back those two top line comments and this looks ready

File: Dockerfile (hunk 7 above)
Comment (drizuid) [reply Dockerfile:1]: LOL i didnt mean to smoke that

**Review by nemchik — APPROVED** [review]

**Pull request merged**